packages/vue-instantsearch/src/util/__tests__/createServerRootMixin.test.js

```diff
@@ -1149,18 +1150,13 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/instantsear
 
       const renderArgs = widget.render.mock.calls[0][0];
 
-      expect(renderArgs).toEqual(
-        expect.objectContaining({
-          state,
-          results,
-          scopedResults: [
-            expect.objectContaining({
-              indexId: 'lol',
-              results,
-            }),
-          ],
-        })
-      );
+      // renders with local state, not the one from results
+      expect(renderArgs.state).toEqual(localState);
+      results._state = localState;
+      expect(renderArgs.results).toEqual(results);
+      expect(renderArgs.scopedResults).toHaveLength(1);
+      expect(renderArgs.scopedResults[0].indexId).toEqual('lol');
+      expect(renderArgs.scopedResults[0].results).toEqual(results);
```

Comment:
Why change to multiple assertions? What didn't work with `expect.objectContaining`?

Reply:
the error message was completely unreadable when the test failed, two massive objects without any diff at all. I had to manually diff to see where the error even was.